ui/tests/integration/helpers/date-from-now-test.js

```diff
@@ -12,6 +12,6 @@ module('Integration | Helper | date-from-now', function(hooks) {
 
   test('you can include a suffix', function(assert) {
     let result = dateFromNow([1481022124443], { addSuffix: true });
-    assert.ok(result.includes(' ago'));
+    assert.ok(result.includes('about'));
```

Comment:
This is a bit of a red flag -- maybe let's double check that the `addSuffix` option is being correctly passed to the update underlying date-fns function

Reply:
It's because the method underneath changed so it changes its wording.  I double-checked on that one.  The old method was deprecated and we moved to formatDistanceToNow, which has is a little smoother with the returns.